### DISCUSSION TITLE
Notebook

### DISCUSSION THREAD

**Discussion by miguelangelrestrepo** [rramosp/ai4eng.v1#201] [00 - General]
![1](https://user-images.githubusercontent.com/105829696/190448733-97bc16f1-ef14-4c1c-81ab-d226047727da.png)

**Profe, en la predición se demora demasiado, y no sabemos si está bueno o malo, me he quedado esperando hasta 4 horas supervisando y eso sigue corriendo... ¿Qué puedo hacer en este caso?

**Comment by Felipe-RA** [comment]
Saludos miguel! 

Disculpa la tardanza en responder!

Puedes probar a correrlo en un entorno local, recomiendo si estas en windows usar [Anaconda](https://docs.anaconda.com/anaconda/install/windows/) o conda si estas en Linux.

También puedes probar a disminuir el tamaño de tu dataset eligiendo filas de forma aleatoria. Esto lo haces para evitar parcializar los datos que tienes.
